src/PublicationModel.js

```diff
@@ -69,30 +69,37 @@ var PublicationModel = Backbone.Model.extend({
 
     // Determine the final attributes, merging in nested objects to prevent overwrite.
     var oldAttributes = _.omit(this.attributes, '_id', 'createdAt');
-    var fullAttributes = ObjectUtils.deepExtend(oldAttributes, newAttributes);
-
-    // Perform the standard Backbone.js set. Do this first, so when we trigger
-    // events further down the new attributes are in place.
+    var fullAttributes = ObjectUtils.deepExtend(ObjectUtils.deepClone(oldAttributes), newAttributes);
+
+    /* Determine which attributes actually changed. This protects against things like the existing
+     * value for an attribute being passed in.  */
+    var changedAttributes = ObjectUtils.changes(oldAttributes, fullAttributes);
+
+    // If there were no real changes, we can bail.
+    var hasChanges = !_.isEmpty(changedAttributes);
+    if (!hasChanges) return;
+
+    /* Perform the standard Backbone.js set. Do this first, so when we trigger events further down
+     * the new attributes are in place. We also determine if all the changes were to nested
+     * objects - if so, we don't emit any events (but still call `set` so the new attributes are
+     * stored on the model).  */
+    var allChangesAreObjects = _.every(changedAttributes, v => _.isObject(v));
```

Comment:
nit: Let's wrap the params for the fat arrow function in parens for readability (so `_.every(changedAttributes, (v) => _.isObject(v));`).